frontend/src/scenes/insights/filters/ActionFilter/entityFilterLogic.ts

```diff
@@ -193,7 +182,10 @@ export const entityFilterLogic = kea<entityFilterLogicType>([
 
             await breakpoint(100)
 
-            actions.loadData(true)
+            const dataLogic = insightDataLogic.findMounted({
+                dashboardItemId: props.typeKey,
+            })
+            dataLogic?.actions?.loadData(true)
```

Comment:
looks safe to me... I guess the worst thing that happens is for some reason its not mounted and someone needs to refresh 👍